torchx/tracker/api.py

```diff
@@ -246,7 +248,7 @@ def run_from_env() -> AppRun:
         this method will retorn same AppRun and hence same instances of the backend implementations.
```

Comment:
would be good to document the ID behavior

Reply:
done